projects/jupyter-server-ydoc/jupyter_server_ydoc/utils.py

```diff
@@ -19,6 +19,7 @@
 class MessageType(IntEnum):
     SYNC = 0
     AWARENESS = 1
+    UPDATE = 2
```

Comment:
I'm not sure `UPDATE` is a good name, since it's similar to a `SYNC_UPDATE` message. Why not just `RAW`, since it's really an unstructured message on top of which we implement a custom protocol?